beancount_reds_importers/libreader/csvreader.py

```diff
@@ -186,7 +189,9 @@ def read_file(self, file):
             rdr = self.prepare_raw_file(rdr)
 
             # extract main table
-            rdr = rdr.skip(getattr(self, "skip_head_rows", 0))  # chop unwanted header rows
+            rdr = rdr.skip(
+                getattr(self, "skip_head_rows", 0)
+            )  # chop unwanted header rows
```

Comment:
Hmm, this line shouldn't have changed. Would you managering have the latest commits which have the right line width for ruff?

Reply:
Yes of course, sorry for that!